app/models/manageiq/providers/nuage/manager_mixin.rb

```diff
@@ -51,20 +146,28 @@ def params_for_create
     # Verify Credentials
     #
     # args: {
+    #   "name" => String,
     #   "endpoints" => {
     #     "default" => {
-    #       "username" => String,
+    #       "default_userid" => String,
+    #       "default_hostname" => String,
+    #       "default_api_port" => Integer,
+    #       "default_security_protocol" => String,
+    #       "password" => String,
+    #     },
+    #     "amqp" => {
```

Comment:
I was expecting we would accept "events" as the key and under the covers we'd handle putting it in the right columns...is that not right?

Reply:
Yeah, we're waiting for the schema change and then we can adjust this :wink: